### ISSUE TITLE
[Feature Request] Show Link Switch to Prevent Punishing/malicious links

### ISSUE THREAD

**Issue by UnixPhonez** [Neutron-Creative/Singlelink#208]
This would be a great security feature for users who are clicking links on one's profile. Basically it's a toggle that can be switched on, on the website, per the user not the owner of the link. To display the full link text of the link behind the buttons. This can prevent punishing links or malicious links from being clicked on, it can also help to report such links. 

How can this be implemented?
At the bottom of the page, there can be a button / switch to "Show Link Text"

Where will the link text be displayed?
The link text will be displayed under the text of the current box

**Comment by jimmybisenius** [comment]
Interesting suggestion. Would this be something a Singlelink admin would toggle, or users could vote/report and would be automated?

ex: Adding a "report page" link to every website, and have a threshold variable set for auto-suspension.

**Comment by UnixPhonez** [comment]
Yes something an site admin can toggle but not the owner of the page, and at the bottom could have a report button but the first offense can be auto suspend by a variable set than set manual review by admin. Just gotta create a system for trolls and bots.
